che/auth.md

```diff
@@ -9,4 +9,18 @@
 * More info on using your own Keycloak instance here https://www.eclipse.org/che/docs/openshift-config.html#multi-user-using-own-keycloak-and-psql
 * Keycloak can be configured (manually) to do User Federation (e.g. LDAP) or Social Login & Brokering if needed, but the default keycloak instance has no Federation or Brokering.
   * https://www.eclipse.org/che/docs/user-management.html
-* A ServiceAccount OAuthClient is *NOT* used by che
\ No newline at end of file
+* A ServiceAccount OAuthClient is *NOT* used by che
+
+# Github Integration
+
+Che has the ability to integrate with Github, providing 2 main features:
+
+* Import a git repo as a Che Project (shows a list of user's repos)
+* Do version control from the Che IDE
+
+There are 2 ways to configure Che server for Github integration.
+
+* Set env vars in Che server for a github client id & secret (embedded/single user mode)
```

Comment:
can you add the env vars names here: CHE_OAUTH_GITHUB_CLIENTID and CHE_OAUTH_GITHUB_CLIENTSECRET?